src/wire/packet_manager.cpp

```diff
@@ -294,45 +294,26 @@ bool PacketManager::HardcodedExecuteFilter(std::string query_type) {
 }
 
 // The Simple Query Protocol
+// Fix mis-split bug
```

Comment:
Anybody reading the code isn't going to know what you are talking about. Please provide a better description